packages/SwingSet/test/test-gc-and-finalize.js

```diff
@@ -22,8 +16,9 @@ function setup() {
     finalized[0] = 'finalizer was called';
   });
   const wr = new WeakRef(victim);
+  const gcAndFinalize = makeGcAndFinalize(engineGC);
```

Comment:
Let's make a separate function to prepare `gcAndFinalize` independently of where we setup the object that's about to be deleted. I'd like the number of objects that go out of scope during the execution of `setup()` to be as small as possible, so we aren't accidentally triggering GC and thus masking a faulty `gcAndFinalize()`.